gcc/plan.sh

```diff
@@ -52,9 +52,7 @@ do_prepare() {
   export LIBRARY_PATH
   build_line "Setting LIBRARY_PATH=$LIBRARY_PATH"
 
-  # TODO: For the wrapper scripts to function correctly, we need the full
-  # path to bash. Until a bash plan is created, we're going to wing this...
-  bash=/bin/bash
+  bash="$(pkg_path_for bash)/bin/bash"
```

Comment:
Counter-intuitively, this just uses the package name I think, not origin/package, at least that is how it is used everywhere else in core-plans.